tket1-passes/README.md

```diff
@@ -7,26 +7,19 @@ API.
 
 ## Building instructions
 
-Building `tket1-passes` is done using `conan >= 2.0`. After installing `conan`,
-(`pip install conan` or use `pipx`), you must set up `conan` to use the
-artifactory containing the tket package:
-
-```
-conan remote add tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs --index 0
-```
-
-For the time being, one extra step is required to export `tket-c-api` to the conan cache:
-
-```
-cd tket-c-api && conan create . --build=missing"
-```
-
-That's it! You can now build the library using `cargo build`.
+Building `tket1-passes` is done using `conan >= 2.0`. This will be installed
+automatically by `uv` if using `just` or `uv`. If you are running `cargo build`
```

Comment:
It is not fully clear to me what the just is referring to, could you maybe give some more details?

Reply:
`just` is a build tool very similar to `make` that is used in `tket2` (see `Justfile` for all the recipes). I've added a reference to `DEVELOPMENT.md` that describes how to build tket2 in more details. Is that ok with you or is there something specific you wanted me to clarify?